index.d.ts

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import * as PIXI from 'pixi.js';
```

Comment:
Forgive my ignorance (I'm not that familiar with TypeScript), but these statements are importing from `@types`, not the actual libraries?

Reply:
Correct - we only need the definitions of these libraries to verify.  I've added the two typings (`@types/react` and `@types/pixi.js`) as `devDependencies` to support this.